<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductItemWidget</class>
 <widget class="QWidget" name="ProductItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblProductName">
     <property name="text">
      <string>Product Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMinus">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spnQuantity"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnPlus">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>Erase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblProductPrice">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
